### PR TITLE
implement SES CloneReceiptRuleSet

### PR DESCRIPTION
**Pull request by wonhyeongseo**
Fix https://github.com/localstack/localstack/issues/6934

@viren-nadkarni May you please guide me to improve? This is my first PR for hacktoberfest. I'm not sure if I've done it correctly. Thank you and I hope you have a wonderful day.

### CODE REVIEW

**Comment by wonhyeongseo** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by wonhyeongseo** [comment]
recheck

File: localstack/services/ses/provider.py [hunk 1]
```diff
@@ -291,3 +293,20 @@ def send_raw_email(
...
+    ) -> CloneReceiptRuleSetResponse:
+        backend = get_ses_backend(context)
+
+        backend.create_receipt_rule_set(rule_set_name)
+        original_rule_set = backend.describe_receipt_rule_set(original_rule_set_name)
+
+        for rule in original_rule_set["Rules"]:
+            backend.create_receipt_rule(rule_set_name, rule)
+
+        return {}
```
Comment (viren-nadkarni) [inline localstack/services/ses/provider.py:312]: You must return an instance of `CloneReceiptRuleSetResponse`

**Review by viren-nadkarni — CHANGES_REQUESTED** [review]
Thank you for contributing. Please take a look at the comment and add some tests. I'll be happy to merge it afterwards.

File: localstack/services/ses/provider.py (hunk 1 above)
Comment (wonhyeongseo) [reply localstack/services/ses/provider.py]: Hello @viren-nadkarni, hope you are doing well.
Thank you for your comment. I changed the response, but I am not sure what to put inside it.

Also I keep falling into this error where it says there is a duplicate rule set. I changed the names and it still doesn't work. May you please help me fix this?
image

Comment (viren-nadkarni) [reply localstack/services/ses/provider.py]: The problem might lie in an earlier error message:

```
list indidces must be integers or slices, not str
```

Might be worth a look?

Comment (wonhyeongseo) [reply localstack/services/ses/provider.py]: Thank you so much @viren-nadkarni !
Your guide was super helpful for figuring the code out 😆 
image

**Comment by wonhyeongseo** [comment]
Hello @viren-nadkarni , may you please tell me if the code needs modifying? Thank you so much for helping :D

**Review by viren-nadkarni — APPROVED** [review]
LGTM :+1: 

@wonhyeongseo Congrats on your first contribution to LocalStack and good luck with Hacktoberfest!

**Pull request merged**